lib/susedistribution.pm

```diff
@@ -571,10 +571,15 @@ sub init_consoles {
             my $packed_ip = gethostbyname($s390_guest_fqdn);
             die "Failed to get host by name for '$s390_guest_fqdn' (on " . hostname . ")" unless $packed_ip;
             my $s390_guest_ip = inet_ntoa($packed_ip);
-            my $host_ip = get_var("AGAMA") ? 'hostip' : 'HostIP';
-            my $host_name = get_var("AGAMA") ? 'hostname' : 'Hostname';
-            $s390_params .= " $host_ip" . "=${s390_guest_ip}/${s390_guest_subnetmask}";
-            $s390_params .= " $host_name" . "=${s390_guest_hostname}";
+            unless (get_var("AGAMA")){
+                $s390_params .= " HostIP=${s390_guest_ip}/${s390_guest_subnetmask}";
+                $s390_params .= " Hostname" . "=${s390_guest_hostname}";
+            }else{
+                my $s390_gateway_ip = "10.145.10.254";
+                my $s390_domain = "oqa.prg2.suse.org";
+                my $s390_device = "enc800";
```

Comment:
all these parameter are dynamic depending if runs in O3 or OSD, should be reflected in openQA variable instead of harcoding them.

Reply:
is it mandatory to add the device?

Reply:
It is added, take a look at this run https://openqa.opensuse.org/tests/4539131#settings 
`ip=10.150.1.152::10.150.1.254:24:openqanet.opensuse.org:enca00:none`

`enca00` is the device in this case.

Reply:
21160037a